File: WAP/W3D5/q3-q4/MyApp/models/product.js
let p = class{
    constructor(name, price, desc,id){
        this.name = name;    
        this.price = price;
        this.desc = desc;
        this.id = id;
    }

    toString(){
        return "id: " + this.id
        + ", Description: " + this.desc
        + ", Price: $" + this.price;
    }
}

module.exports = p;

File: EA/Lab3-Sujit-615185/Lab3PartA/src/main/java/customers/Product.java
package customers;

public class Product {
    private int productNumber;
    private  String name;
    private String price;

    private Supplier supplier;

    public Product(int productNumber, String name, String price) {
        this.productNumber = productNumber;
        this.name = name;
        this.price = price;
    }

    public int getProductNumber() {
        return productNumber;
    }

    public void setProductNumber(int productNumber) {
        this.productNumber = productNumber;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPrice() {
        return price;
    }

    public void setPrice(String price) {
        this.price = price;
    }

    @Override
    public String toString() {
        return "Product{" +  "productNumber=" + productNumber + ", name='" + name + '\'' + ", price='" + price + '\'' + '}';
    }
}
